elements/rhelement/test/rhelement_cascade_attribute_test.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+    <script src="/components/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="/components/web-component-tester/browser.js"></script>
+  </head>
+  <body>
+
+    <rh-parent test-attr="foo"></rh-parent>
+
+    <script type="module">
+      import RHElement from "../rhelement.js";
+
+      class RhParent extends RHElement {
+        static get tag() {
+          return "rh-parent"
+        }
+
+        static get observedAttributes() {
+          return ['test-attr'];
+        }
+
+        static get cascadingAttributes() {
+          return {
+            "test-attr": "rh-child"
+          };
+        }
+
+        get html() {
+          return `
+            <rh-child id="rh-child-1">rh-child 1</rh-child>
+            <rh-child id="rh-child-2">rh-child 2</rh-child>
+            <div id="no-cascade">div div div div</div>
+          `;
+        }
+
+        constructor() {
+          super(RhParent);
+        }
+      }
+
+      RHElement.create(RhParent);
+
+      class RhChild extends RHElement {
+        static get tag() {
+          return "rh-child";
+        }
+
+        get html() {
+          return `<slot></slot>`;
+        }
+
+        constructor() {
+          super(RhChild);
+        }
+      }
+
+      RHElement.create(RhChild);
+
+      suite('<rhelement>', () => {
+
+        const testAttrName = 'test-attr';
+
+        test("it should cascade values of cascadingAttributes", () => {
+          const parent = document.querySelector("rh-parent");
+
+          const child1 = parent.shadowRoot.querySelector('#rh-child-1');
+          const child2 = parent.shadowRoot.querySelector('#rh-child-2');
+
+          const attrVal = parent.getAttribute(testAttrName);
+
+          assert.equal(attrVal, child1.getAttribute(testAttrName));
+          assert.equal(attrVal, child2.getAttribute(testAttrName));
+        });
+        test("it should cascade changes to cascadingAttributes", () => {
+          const parent = document.querySelector("rh-parent");
+
+          const child1 = parent.shadowRoot.querySelector('#rh-child-1');
+          const child2 = parent.shadowRoot.querySelector('#rh-child-2');
+
+          const attrVal = "new-val";
+
+          parent.setAttribute(testAttrName, attrVal);
+
+          assert.equal(attrVal, child1.getAttribute(testAttrName));
+          assert.equal(attrVal, child2.getAttribute(testAttrName));
+        });
+        test("it should cascade values of cascadingAttributes to unspecified elements ", () => {
```

Comment:
@mwcz - Should this be "it should **not** cascade values..."?

Reply:
Yeah, nice catch, pushed a fix.